Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnUpdate.hs

```diff
@@ -433,7 +433,9 @@ onUpdate ValidatedBookingCancelledReq {..} = do
   let bookingCancellationReason = mkBookingCancellationReason booking.id (mbRide <&> (.id)) cancellationSource booking.merchantId
   merchantConfigs <- CMC.findAllByMerchantOperatingCityId booking.merchantOperatingCityId
   case cancellationSource of
-    SBCR.ByUser -> SMC.updateCustomerFraudCounters booking.riderId merchantConfigs
+    SBCR.ByUser -> case booking.bookingDetails of
```

Comment:
This should not depend on rentals. It has been fixed properly by Piyush.
https://github.com/nammayatri/nammayatri/pull/5494/files